docs/javascript/v2/release-notes/release-notes.mdx

```diff
@@ -16,6 +16,21 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hms-virtual-background      | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-virtual-background.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-virtual-background)           |
 
 
+## 28-11-2024
+
+Released: `@100mslive/hms-video-store@0.12.25`, `@100mslive/react-sdk@0.10.25`, `@100mslive/hls-player@0.3.25`, `@100mslive/roomkit-react@0.3.25`, `@100mslive/hms-whiteboard@0.0.15`, `@100mslive/hms-virtual-background@1.13.25`
+
+### Breaking
+
+- Implemented internal method to check if recording should continue when no active elements are present
```

Comment:
This is not a breaking change and does not need to be mentioned. This will be used by beam only

Reply:
Done